html-block-and-inline/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>HTML Block and Inline</title>
+  </head>
+  <body>
+    <h1>Blue Americana Snuggie&#174; Blanket</h1>
+    <h2>by SnuggieStore</h2>
+    <p>Price: <s>$19.99</s> $14.99</p>
+    <p>
+      Part of our new Americana collection, this <em>beautiful</em> blue is
```

Comment:
We mentioned it in Q&A while you were gone, but in this case, using something like the `<strong>` element would be a better choice because it's more semantic. We want to take advantage of semantic elements as much as possible for their SEO & accessibility benefits